environment/src/test/java/com/sequenceiq/environment/environment/poller/SdxPollerProviderTest.java

```diff
@@ -0,0 +1,93 @@
+package com.sequenceiq.environment.environment.poller;
+
+import static java.util.Collections.emptyList;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.mockito.Mockito;
+
+import com.dyngr.core.AttemptResult;
+import com.dyngr.core.AttemptState;
+import com.sequenceiq.sdx.api.endpoint.SdxEndpoint;
+import com.sequenceiq.sdx.api.model.SdxClusterResponse;
+import com.sequenceiq.sdx.api.model.SdxClusterStatusResponse;
+
+public class SdxPollerProviderTest {
```

Comment:
JUnit5 does not need public class and public methods. please make them package-private